### PR TITLE
Adding Prettier to CodeBase

### PR DESCRIPTION
**Pull request by ravjot07**
contributes towards #876 adds prettier to code base

### CODE REVIEW

**Comment by ravjot07** [comment]
@aaronreed708 should i format code by running this command in this PR like what should be my approach..... for formatting the the code......

File: code/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "tabWidth": 4,
+    "useTabs": false,
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "es5",
+    "bracketSpacing": true,
+    "jsxBracketSameLine": false,
+    "arrowParens": "always",
+    "endOfLine": "lf"
+  }
```
Comment (aaronreed708) [inline code/.prettierrc:12]: It looks like jsxBracketSameLine has been deprecated in favor of bracketSameLine.

**Review by aaronreed708 — APPROVED** [review]
I think I am good with all of these options.  I noted a deprecation that needs to be fixed.

**Comment by aaronreed708** [comment]
> @aaronreed708 should i format code by running this command in this PR like what should be my approach..... for formatting the the code......

I think that before this PR is merged, I'd like to see another PR that shows what the code will look like after Prettier runs.  I really only want to do this massive merge once.  I think I'd like to keep the PRs separate so that it is very clear when the Prettier changes dropped and not hide it amongst the massive code changes that will happen.  This PR should be merged before the second PR is merged, of course.

Do you think that this approach makes sense?  Or am I being too cautious?

**Comment by ravjot07** [comment]
> > @aaronreed708 should i format code by running this command in this PR like what should be my approach..... for formatting the the code......
> 
> I think that before this PR is merged, I'd like to see another PR that shows what the code will look like after Prettier runs. I really only want to do this massive merge once. I think I'd like to keep the PRs separate so that it is very clear when the Prettier changes dropped and not hide it amongst the massive code changes that will happen. This PR should be merged before the second PR is merged, of course.
> 
> Do you think that this approach makes sense? Or am I being too cautious?

i liked Your approach sir we should be cautious before merging this big code formatting's

**Comment by aaronreed708** [comment]
Thank you for your work on this issue @ravjot07!  It was the foundation for the work in https://github.com/finos/a11y-theme-builder/pull/948.  Closing this PR as complete.

**Pull request closed without merging**